forge.lic

```diff
@@ -293,19 +294,42 @@ class Forge
   end
 
   def add_fuel
-    stow_item('tongs')
```

Comment:
I wonder if the code would be any clearer if you wrapped the `stow` and `get` here with an `if @articulated` check. Something like:

```
if @articulated
  # do new stuff
else
  # do old stuff
end

# push fuel like the old code

if @articulated
  # put away with new code
else
  # put away with old code
end
```

Reply:
Alright, I'll work on doing that and testing it today